### ISSUE TITLE
Failed to start the project

### ISSUE THREAD

**Issue by dadiorchen** [Greenstand/treetracker-wallet-web#106]
**Describe the bug**
`npm start` failed

**To Reproduce**
`npm start`

get error:

```
treetracker-wallet-web> npm start

> wallet-app@0.1.0 start /Users/deanchen/work/greenstand/treetracker-wallet-web
> react-scripts start

internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module 'webpack'
Require stack:
- /Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/terser-webpack-plugin/dist/index.js
- /Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/terser-webpack-plugin/dist/cjs.js
- /Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/react-scripts/config/webpack.config.js
- /Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/react-scripts/scripts/start.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/terser-webpack-pl
ugin/dist/index.js:14:40)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/terser-webpack-plugin/dist/index.js',
    '/Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/terser-webpack-plugin/dist/cjs.js',
    '/Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/react-scripts/config/webpack.config.js'
,
    '/Users/deanchen/work/greenstand/treetracker-wallet-web/node_modules/react-scripts/scripts/start.js'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wallet-app@0.1.0 start: `react-scripts start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wallet-app@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/deanchen/.npm/_logs/2021-09-14T07_32_58_618Z-debug.log
treetracker-wallet-web>

```

**Comment by dadiorchen** [comment]
I guess we are merged some problematic PRs

**Comment by dadiorchen** [comment]
This thread is talking about global webpack, but I suppose it is a bad practice to rely on global npm:
https://stackoverflow.com/questions/29492240/error-cannot-find-module-webpack

**Comment by YassinHussein** [comment]
I faced the same issue yesterday 
when run `npm start ` 

removing node_modules and package-lock.json  and run `npm i` fixed the issue for me

**Comment by dadiorchen** [comment]
The same error message? @YassinHussein

**Comment by YassinHussein** [comment]
yes the same error message

**Comment by dadiorchen** [comment]
@YassinHussein thanks
@vishal5251 forgot it, I have no problem with this error now, thanks!

resolved by #108
